crates/core/src/contracts/account.rs

```diff
@@ -43,7 +43,12 @@ pub trait Account<'a, P: Provider + Send + Sync + 'a> {
             _ => Err(EthApiError::from(err)),
         })?;
 
-        Ok(vec_felt_to_bytes(bytecode))
+        if bytecode.is_empty() {
+            return Ok(Bytes::default());
+        }
+
+        // bytecode_len is the first element of the returned array
+        Ok(vec_felt_to_bytes(bytecode[1..].to_vec()))
```

Comment:
i suggest adding a marker for where we do "manual decoding" in our RPC codebase, so that the day we have generic decoding, we can replace in all places

Reply:
```suggestion
        // TODO: Remove Manual Decoding
        Ok(vec_felt_to_bytes(bytecode[1..].to_vec()))
```